examples/cloud-router-2-virtual-device-connection/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.4"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ">= 1.20.0"
```

Comment:
Let's make this >= 1.31.0. Keep it to the maximum starting version when we add a new example. Keeps from possible bugs being exposed to the example in earlier versions.